### PR TITLE
Customize contact page [PLAT-34394]

### PR DESCRIPTION
**Pull request by nc-sjoquistn**

### CODE REVIEW

File: scripts/config/design-object-config.js [hunk 1]
```diff
@@ -7,3 +7,22 @@ _.merge(window.designObjects, { // jshint ignore:line
     }
   }
 });
+
+const configOverrides = _.merge({
+  contactConfig: {
+    paragraph: {
+      getInfo({ interpolate, recordKeeper } = {}) {
```
Comment (hrjr4433) [inline scripts/config/design-object-config.js]: hmmm there is [change](https://github.com/BLC/nextcapital-design-objects/pull/1437/files#diff-2f4a63942dc2b04f158f0a040b9b9a56) coming

Comment (nc-sjoquistn) [reply scripts/config/design-object-config.js]: noooo

Comment (nc-sjoquistn) [reply scripts/config/design-object-config.js]: well....YESSSS! that contact component code is HORRIBLE

Comment (nc-sjoquistn) [reply scripts/config/design-object-config.js]: but this needs to be in by tomorrow

**Pull request merged**